lib/src/models/tag.dart

```diff
@@ -10,10 +10,11 @@ class Tag {
   /// The tag's value encoded as Base64.
   final String value;
 
-  // TODO: Verify tag size.
-  // TODO: Move encoding logic into [Tag] class while ensuring it does not break JSON decoding.
   Tag(this.name, this.value);
 
   factory Tag.fromJson(Map<String, dynamic> json) => _$TagFromJson(json);
   Map<String, dynamic> toJson() => _$TagToJson(this);
+
+  @override
+  bool operator ==(o) => o is Tag && name == o.name && value == o.value;
```

Comment:
I wonder how the operator override looks like after it's compiled into JS :thinking: